overlord/ifacestate/handlers.go

```diff
@@ -336,6 +333,26 @@ func (m *InterfaceManager) undoDiscardConns(task *state.Task, _ *tomb.Tomb) erro
 	return nil
 }
 
+func getDynamicHookAttributes(task *state.Task) (map[string]interface{}, map[string]interface{}, error) {
+	var plugAttrs, slotAttrs map[string]interface{}
+
+	var err error
+	if err = task.Get("plug-dynamic", &plugAttrs); err != nil {
```

Comment:
Nitpick: You can drop `var err error` above and just `:=` assign the err variable here.